src/Game.re

```diff
@@ -0,0 +1,36 @@
+type env =
+  | Sand
+  | Spice
+  | HighSpice
+  | Rocks
+  | Mountains;
+
+let createPlayingBoard = (width, height) =>
```

Comment:
named arguments will be more descriptive, maybe: `let createPlayingBoard = (~width, ~height) =>`